onnxscript/function_libs/torch_lib/ops/core.py

```diff
@@ -7698,6 +7698,21 @@ def aten_sinh(self: TFloat) -> TFloat:
     return op.Sinh(self)
 
 
+@torch_op(("aten::slice.Tensor"), trace_only=True, complex=True)
+def aten_slice_complex(
+    self: TTensor,
+    dim: int = 0,
+    start: Optional[INT64] = None,
+    end: Optional[INT64] = None,
+    step: Optional[INT64] = None,
+) -> TTensor:
+    """slice.Tensor(Tensor(a) self, int dim=0, SymInt? start=None, SymInt? end=None, SymInt step=1) -> Tensor(a)"""
+    if dim < 0:
+        # Account for the complex dimension in ONNX
+        dim = dim - 1
```

Comment:
I think this could be a little confusing for users when they examine the graph. What do you think if we did `dim += len(self.shape)`?

Reply:
OK I can create a follow up

Reply:
https://github.com/microsoft/onnxscript/pull/2094